doc/sphinx/language/core/conversion.rst

```diff
@@ -6,6 +6,8 @@ Conversion rules
 In |Cic|, there is an internal reduction mechanism. In particular, it
 can decide if two programs are *intentionally* equal (one says
 :term:`convertible`). Convertibility is described in this section.
+See :ref:`controlling-the-reduction-strategies` describes how
+to influence how convertibility is determined.
```

Comment:
No. AFAIK controlling the reduction strategies won't have a critical influence on how the kernel eventually decide if two terms are convertible. I was rather suggesting linking to the "Performing computations" section for how to apply each of the reduction rules described below.